skfuzzy/__init__.py

```diff
@@ -62,50 +62,68 @@
 
 # Enable testing of the package
 import os.path as osp
-import imp
 import functools
 import warnings
 import sys
 
 pkg_dir = osp.abspath(osp.dirname(__file__))
 data_dir = osp.join(pkg_dir, 'data')
+nose_installed = False
```

Comment:
We assume `nose` is not installed by default, and then only try to import for Py3 and Py2. Upon success, we set this flag to `True`. Then later we use this flag to choose which function to use.